app/src/main/java/me/hufman/androidautoidrive/music/spotify/SpotifyWebApi.kt

```diff
@@ -34,6 +37,7 @@ class SpotifyWebApi private constructor(val context: Context, val appSettings: M
 		const val TAG = "SpotifyWebApi"
 		const val NOTIFICATION_CHANNEL_ID = "SpotifyAuthorization"
 		const val NOTIFICATION_REQ_ID = 56
+		const val LIKED_SONGS_PLAYLIST_NAME = "ANDROIDAUTOIDRIVE_LIKED_SONGS"
```

Comment:
I'm planning to rename the project soon for 1.3.1, can this be preemptively changed to `AAIDRIVE_LIKED_SONGS`?

Reply:
Yeah, updated to "AAIDRIVE_LIKED_SONGS"